test/Microsoft.ML.StaticPipelineTesting/StaticPipeTests.cs

```diff
@@ -254,5 +258,74 @@ public void StaticPipeAssertKeys()
                    stay: c.KeyU4.TextValues.Scalar,
                    alot: c.KeyU1.I4Values.Vector)));
         }
+
+        [Fact]
+        public void Normalizer()
```

Comment:
>Normalizer [](start = 20, length = 10)

Maybe we should instead move that to `Microsoft.ML.Tests.Transformers` and join with other normalizer tests?

Reply:
That would make sense to me. I'd rather do reorganizations later though if that's all right, not in this PR?

---
In reply to: [215748768](https://github.com/dotnet/machinelearning/pull/843#discussion_r215748768) [](ancestors = 215748768)